src/test/kotlin/org/rust/ide/intentions/RustIntentionsTest.kt

```diff
@@ -21,6 +21,18 @@ class RustIntentionsTest : RustTestCaseBase() {
         PlatformTestUtil.assertDirectoriesEqual(afterDir, beforeDir)
     }
 
+    fun testContractModule() {
+        val before = testName + "/before"
+        var after = testName + "/after"
```

Comment:
String interpolation is preferred over concatenating strings in Kotlin. It's better to use `"$testName/before"` instead of `testName + "/before"`.